samples/secure-with-tls/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  eventstore:
+    image: docker.pkg.github.com/eventstore/eventstore/eventstore:ci
```

Comment:
I think we should reference the official containers on dockerhub, as at the moment you need to create an API token and use `docker login` to pull our CI packages. However If we move to github _container_ registry that problem goes away.

Reply:
Fixed.